asset_dashboard/models.py

```diff
@@ -420,6 +420,10 @@ class FundingStream(models.Model):
         null=True,
     )
 
+    def save(self, *args, **kwargs):
+        self.full_clean()
+        return super().save(*args, **kwargs)
```

Comment:
were you not able to get the validation to work without this? i'm wondering if `full_clean` is already called whenever the form validation happens?

Reply:
oh you're right! i think i was having issues with validation not running when i was applying validators directly to the models, and this might have helped?

but regardless, i'm doing it through the serializer now and it works without this save override, so i've removed this